source/elements/oneMKL/source/domains/spblas/operations/spmm.rst

```diff
@@ -302,7 +304,9 @@ spmm
 
    A_view
       Specifies which part of the handle should be read as described by
-      :ref:`onemkl_sparse_matrix_view`.
+      :ref:`onemkl_sparse_matrix_view`. The ``type_view`` field must be
+      ``matrix_descr::general`` and the ``uplo_view`` and ``diag_view`` fields
+      are ignored.
```

Comment:
```suggestion
      :ref:`onemkl_sparse_matrix_view`. The ``type_view`` field must be
      ``matrix_descr::general`` and the ``uplo_view`` and ``diag_view`` fields
      are currently ignored, but support may be expanded in the future.
```

Reply:
I have answered this in the comment above https://github.com/uxlfoundation/oneAPI-spec/pull/542#discussion_r1723195090